test/parallel/test-cli-node-options.js

```diff
@@ -34,7 +34,7 @@ if (common.hasCrypto) {
 expect('--abort_on-uncaught_exception', 'B\n');
 expect('--max-old-space-size=0', 'B\n');
 expect('--stack-trace-limit=100',
-       /(\s*at f \(\[eval\]:1:\d*\)\n){100}/,
+       /(\s*at f \(\[eval\]:1:\d*\)\r?\n){100}/,
```

Comment:
Maybe worth making it a non-capturing group:

```diff
-        /(\s*at f \(\[eval\]:1:\d*\)\r?\n){100}/, 
+        /(\s*at f \(?:\[eval\]:1:\d*\)\r?\n){100}/, 
```

Reply:
Or rather
```dif
-        /(\s*at f \(\[eval\]:1:\d*\)\r?\n){100}/, 
+        /(?:\s*at f \(\[eval\]:1:\d*\)\r?\n){100}/, 
```

Reply:
Since this is a test file, I’d go with clarity over performance? I know it took me a while to get the hang of what `?:` means when learning regexes and seeing that in the wild…

Reply:
Interesting, I find not using non-capturing groups more confusing, as for me a group is for capturing (I think using it for capturing is more common than using it to repeat with `(){x}`).

The reason I suggested it was because my first thought was "why does the test need to capture 100 different groups", and the fact that the regex gets passed through a custom function makes it harder to track. Performance is a secondary concern.